src/Symfony/Component/Messenger/composer.json

```diff
@@ -21,6 +21,7 @@
     },
     "require-dev": {
         "psr/cache": "^1.0|^2.0|^3.0",
+        "symfony/clock": "^6.3",
```

Comment:
It's not possible to construct a worker without a clock, which makes `symfony/clock` a mandatory dependency.

Reply:
fixed